modules/ui.py

```diff
@@ -69,12 +69,6 @@ def plaintext_to_html(text):
 
 
 def image_from_url_text(filedata):
-    if type(filedata) == list:
-        if len(filedata) == 0:
-            return None
-
-        filedata = filedata[0]
-

```

Comment:
This function is named `image_from_url_text`; nothing should be sending it a list.